pypresence/baseclient.py

```diff
@@ -13,11 +13,8 @@
 
 class BaseClient:
 
-    def __init__(self, client_id: str, **kwargs):
-        pipe = kwargs.get('pipe', 0)
-        loop = kwargs.get('loop', None)
-        handler = kwargs.get('handler', None)
-        self.isasync = kwargs.get('isasync', False)
```

Comment:
Not sure why you think these are better suited as predefined kwargs

Reply:
Because then the developer can read the function documentation instead of the online documentation. It improves productivity.

Reply:
My IDE seems to understand exactly what kwargs are supposed to be passed just fine. Not too bothered about supporting people who write code in IDLE.

Reply:
![image](https://user-images.githubusercontent.com/21298211/90548589-2c50fc80-e15b-11ea-8e22-3e26e8fcc062.png)
Using PyCharm BTW not IDLE. It's impossible to use functions if they use args and kwargs and then don't document what args/kwargs are supposed to be in the source code.